Engine/World.cs

```diff
@@ -64,17 +96,23 @@ public IEnumerable<Entity> Entities
         /// Collider of the world (the walls).
         /// </summary>
         public WorldCollider Collider { get; private set; }
-        
+
+        /// <summary>
+        /// Stores entities into a region with all other entities in the same region.
+        /// </summary>
+        public List<Entity>[][] Regions
+        {
+            get => _regions;
+            private init => _regions = value;
+        }
```

Comment:
auto property

Reply:
Same as https://github.com/PolyNoradrenalin/AntSimulator/pull/113#discussion_r762564488
Personal preference to use the more verbose version